### PR TITLE
Update certa dependency so that ARM processors are supported

### PR DESCRIPTION
**Pull request by anmolshres98**

### CODE REVIEW

File: common/changes/@itwin/oidc-signin-tool/as-update-certa-dependency_2022-08-06-00-15.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/oidc-signin-tool",
+      "comment": "Update certa to version 3.4.0-dev.3",
+      "type": "patch"
```
Comment (aruniverse) [inline common/changes/@itwin/oidc-signin-tool/as-update-certa-dependency_2022-08-06-00-15.json:6]: I'm thinking minor bump for this? And then patch when the official 3.4 release is out?

**Review by aruniverse — APPROVED** [review]

File: common/changes/@itwin/oidc-signin-tool/as-update-certa-dependency_2022-08-06-00-15.json (hunk 1 above)
Comment (anmolshres98) [reply common/changes/@itwin/oidc-signin-tool/as-update-certa-dependency_2022-08-06-00-15.json]: That's alright by me

File: packages/oidc-signin-tool/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@itwin/certa": "^3.0.0",
+    "@itwin/certa": "3.4.0-dev.3",
```
Comment (calebmshafer) [inline packages/oidc-signin-tool/package.json:35]: I don't think we should lock down the specific version here. Can we backport the change to 3.3.0 instead?

**Review by calebmshafer — CHANGES_REQUESTED** [review]

File: packages/oidc-signin-tool/package.json (hunk 2 above)
Comment (anmolshres98) [reply packages/oidc-signin-tool/package.json:35]: Gothcha, just made a [PR}(https://github.com/iTwin/itwinjs-core/pull/4039) for the backport

Comment (calebmshafer) [reply packages/oidc-signin-tool/package.json:35]: Agreed, however I wasn't sure about bumping this as a minor version and then patching it to move to the new minor release.

Comment (aruniverse) [reply packages/oidc-signin-tool/package.json:35]: Can we break locktstep for these tooling pkgs like certa?

Comment (calebmshafer) [reply packages/oidc-signin-tool/package.json:35]: we should, let's add it to the backlog.

**Comment by anmolshres98** [comment]
Drafting till 3.3 release of core

**Comment by anmolshres98** [comment]
@aruniverse @calebmshafer should be ready for review now

**Comment by aruniverse** [comment]
I dont think we need this pr, if it's going to automatically pull in certa@^3.0, which should pull in 3.3.0

**Comment by aruniverse** [comment]
closing as is we use ^3.0.0 in oidc-client-tool so should pull in the latest ver of puppeteer

**Pull request closed without merging**